src/presentation/response_models.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel, create_model
+from typing import Union
+import os
+
+
+class CrossMatchModel(BaseModel):
+    attribute_name: str
+    unit: str
+    value: Union[float, None]
```

Comment:
Esto funciona, pero por una cuestión de compatibilidad a futuro sugeriría usar:
```suggestion
    value: float | None
```
Esto es general para todos los usos de `Union`

Reply:
Y una duda adicional:

Cuando hacen la limpieza, los infinitos los convierten a string... ese caso debería fallar para este modelo

Reply:
Gracias por la sugerencia, ahora en donde se utilizaban los `Union` se reemplazo por `value: float | str | None`